bamboo/install_pyenv.sh

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env bash
```

Comment:
This is a helper script for installing pyenv from https://github.com/pyenv/pyenv-installer/blob/master/bin/pyenv-installer @c5475539c464fa167406d7103ca92f09a5445ddb.   Including in source as an alternative to piping via bash.